README.md

```diff
@@ -27,41 +27,28 @@ Base requirements:
 * Python 3
 * Redis
 * PostgreSQL
-* gulp
 
-## Linux
+## On Linux
```

Comment:
Should we explicitly mention Mac OS?

Reply:
I have mac and it works just fine with the Linux instructions. But I guess we can see about this later.